src/EventBus/EventBridgeClient.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace RL\EventBus;
+
+use Aws\Exception\AwsException;
+use Aws\EventBridge\EventBridgeClient as BaseEventBridgeClient;
+use Psr\Log\LoggerInterface;
+
+class EventBridgeClient
+{
+    /** @var BaseEventBridgeClient */
+    private BaseEventBridgeClient $client;
+
+    /** @var LoggerInterface */
+    private LoggerInterface $logger;
+
+    /** @var string */
+    private string $env;
+
+    public function __construct(BaseEventBridgeClient $client, LoggerInterface $logger, string $env)
+    {
+        $this->client = $client;
+        $this->logger = $logger;
+        $this->env = $env;
+    }
+
+    public function putEvent(int $tenant, string $name, string $detail)
+    {
+        try {
+            $result = $this->client->putEvents([
+                'Entries' => [
+                    [
+                        'Detail' => $detail,
+                        'DetailType' => $name,
+                        'EventBusName' => $this->getEventBusName(),
+                        'Resources' => ['apps/' . $tenant],
+                        'Source' => $this->env,
```

Comment:
This should be the application name. We need another parameter in the constructor named `$source` and it should be set from the Symfont env variable `APP_NAME`